index.bs

```diff
@@ -680,6 +698,15 @@ To <dfn>trigger attribution</dfn> given an [=attribution trigger=] |trigger| run
     [=attribution report/attribution destination=] equals |attributionDestination|.
 1. If |numMatchingReports| is greater than or equal to the user agent's
     [=max reports per attribution destination=], return.
+1. Set |rateLimitCutoff| to an [=implementation-defined=] time before |trigger|'s [=attribution trigger/trigger time=].
+1. Let |sourceSite| be the result of running [=parse a site=] with |sourceToAttribute|'s [=attribution source/source origin=].
+1. If |sourceSite| is null, return.
+1. Let |matchingRateLimitRecords| be all entries in the [=attribution rate-limit cache=] where all of the following are true:
+     * entry's [=attribution rate-limit record/source site=] and |sourceSite| are equal
+     * entry's [=attribution rate-limit record/attribution destination=] and |attributionDestination| are equal
+     * entry's [=attribution rate-limit record/source type=] and |sourceToAttribute|'s [=attribution source/source type=] are equal
+     * entry's [=attribution rate-limit record/trigger time=] is greater than |rateLimitCutoff|
```

Comment:
rateLimitTimePeriod / rateLimitWIndowLength or something might make this variable a bit more clear